.github/workflows/automatic-updates.yml

```diff
@@ -43,8 +43,7 @@ jobs:
       - name: Auto-approve the PR
```

Comment:
I'm thinking this and the auto-merge should probably be disabled till we figure out not including the unrelated key changes

Reply:
fair, separate from changing credentials used when creating the PR, tho

Reply:
actually, if the key change for whatever reason is a blocker, then the entire PR is useless since you wanna change it anyways